ibis/util.py

```diff
@@ -555,3 +557,8 @@ def _absolufy_paths(name):
 
     source = _absolufy_paths(source)
     return source
+
+
+def generate_unique_table_name(namespace: str) -> str:
+    """Creates case-insensitive uuid4 unique table name."""
+    return f"_ibis_{namespace}_{np.base_repr(uuid.uuid4().int, 36)}".lower()
```

Comment:
I'll adjust `ibis.util.guid()` to use the base 36 representation for more compact output. Just a note-to-self for me here.